app/src/main/java/org/falaeapp/falae/fragment/PageFragment.kt

```diff
@@ -116,6 +117,18 @@ class PageFragment : Fragment(), ViewPagerItemFragment.PageInteractionListener {
         }
     }
 
+    private fun handleFragmentLifeCycle(position: Int) {
+        val fragmentToShow = mPagerAdapter.getItem(position)
+        fragmentToShow.userVisibleHint = true
+        (fragmentToShow as FragmentLifecycle).onResumeFragment()
+
+        val fragmentToHide = mPagerAdapter.getItem(itemCurrentPosition)
+        userVisibleHint = false
```

Comment:
Although I'm able to understand it, I think it's worth to add a comment around what's being achieved here. `userVisibleHint` fragment attribute vs `fragmentToShow.userVisibleHint`, `onResumeFragment()` vs `onPauseFragment()`.